data/distro.yml

```
@@ -542,3 +542,34 @@
       text: '
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
         <p>Note: graphical installation of Flatpak apps may not be possible with GNU Guix.</p>'
+      </li>
+    </ol>
+
+- name: "Crystal Linux"
+  logo: "crystallinux.svg"
+  info: >
+    <ol class="distrotut">
+      <h2>Flatpak is installed by default on Crystal Linux. However, if you didn't use jade_gui to install crystal or selected not to install it, you can set Flatpak up by using the following steps.<h3>
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">$</span> ame upg
+        </code></pre>
+        <p>Then install Flatpak:</p>
+          <pre><code>
+          <span class="unselectable">$</span> ame ins flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+      </li>
+    </ol>
```

```suggestion
- name: Crystal Linux
  logo: "crystallinux.svg"
  introduction: >
      <h2>Flatpak is installed by default on Crystal Linux.<h2>
      <p>If you didn't use jade_gui to install crystal or selected not to install it, you can set Flatpak up by using the following steps.<p>
  steps:
    - name: Install Flatpak
      text:
        "<p>To install Flatpak in Crystal Linux, you must first make sure your packages are up to date, run the following in a terminal:</p>
        <terminal-command>ame upg</terminal-command>
        <p>Then install Flatpak:</p>
        <terminal-command>ame ins flatpak</terminal-command>"
    - name: Add the Flathub repository
      text:
        "<p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
        <terminal-command>flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo</terminal-command>"
    - name: Restart
      text:
        "<p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>"
```

I forgot that it also needs to be rewritten into a new format.

---

@axtloss I will merge the PR after these changes are applied.

---

@axtloss Ping. :-)